app/models/course.rb

```diff
@@ -570,6 +578,44 @@ def has_the_modern_languages_secondary_subject_type?
     subjects.any? { |subject| subject&.id == SecondarySubject.modern_languages.id }
   end
 
+  def validate_subject_count
+    return if subjects.empty?
+
+    case level
+    when "primary", "further_education"
+      if subjects.count > 1
+        errors.add(:subjects, "has too many subjects")
+      end
+    when "secondary"
+      if subjects.count > 2 && !has_any_modern_language_subject_type?
```

Comment:
is it out of scope for `modern_language`

`modern_language` + `French` + `Biology`

Reply:
Asked @fofr

Reply:
@fofr has said that we don't need to restrict this atm